docs/usage.rst

```diff
@@ -22,6 +22,43 @@ will render to this:
 This will :hoverxref:`show a tooltip <hoverxref:hoverxref>` in the linked words to ``hoverxref``.
 
 
+Tooltip on intersphinx content
+------------------------------
+
+Sphinx comes with a nice built-in extension called :doc:`sphinx.ext.intersphinx <sphinx:usage/extensions/intersphinx>`
+that allows to generate links to the documentation of sections in other projects.
+
+You can combine this extension with ``sphinx-hoverxref`` to show tooltips over these links to other projects.
+For example, this documentation itself configures intersphinx with Read the Docs documentation and allow us
+to do the following:
+
+.. code-block:: rst
+
+   Show a tooltip for :doc:`Read the Docs automation rules <readthedocs:automation-rules>`.
+
+That will render to:
+
+Show a tooltip for :doc:`Read the Docs automation rules <readthedocs:automation-rules>`.
```

Comment:
```suggestion
   Show a tooltip for :doc:`Read the Docs automation rules <readthedocs:/automation-rules>`.

That will render to:

Show a tooltip for :doc:`Read the Docs automation rules <readthedocs:/automation-rules>`.
```

I think we want this to be absolute? It must default to /, but being explicit is good

Reply:
It doesn't work

```
WARNING: unknown document: readthedocs:/automation-rules
```

I think the `/` is allowed when using `:doc:` but the identifiers for intersphinx do not allow them because is not a path? 🤔

Reply:
Huh, odd. Learn something new everyday!